site/specification/ORCv1.md

```diff
@@ -1297,7 +1297,7 @@ in a bloom filter is as follows:
   * position = combinedHash % m
 6. Set the position in bit set. The LSB 6 bits identifies the long index
    within bitset and bit position within the long uses little endian order.
-  * bitset[position >>> 6] \|= (1L << position);
+  * bitset[position >> 6] \|= (1L << (position & 0x3f));
```

Comment:
Thank you for making a PR, @dirtysalt . Could you add the URL to the corresponding code part into your PR description, please?

Reply:
Thanks.  Here is the corresponding code https://github.com/apache/orc/blob/master/c%2B%2B/src/BloomFilter.cc#L47. 

The code is correct, but the documentation is not. The code is to use `% 64` which has the same effect as `& 0x3f`. 

```
  constexpr uint64_t BITS_OF_LONG = 64;
  constexpr uint8_t  SHIFT_6_BITS = 6;
  void BitSet::set(uint64_t index) {
    mData[index >> SHIFT_6_BITS] |= (1ULL << (index % BITS_OF_LONG));
  }
```

Reply:
Thanks. Sure, I have updated the PR description and title.  Do you think it's clear enough?

Reply:
Thank you for update. Can we use `% 64` consistently instead of `& 0x3f`? At line 1297, we already use `%`.

Reply:
Yes. I've updated. It's much more consistent.